.gitignore

```diff
@@ -40,3 +40,9 @@ dump.rdb
 
 # SAML certificates generated by ./tools/generate-ssl-certs.sh
 certs/
+
+# gastby ,cache directory
+.cache/
```

Comment:
These folders will be under `src/frontend` right?  We should prefix

Reply:
yes these are under frontend. By prefix do you mean writing out the full path: `./src/frontend/.cache/` ?

Reply:
Right, exactly.  Otherwise it's going to ignore them in the root, where they don't exist.